src/components/ResumeUploadBtn.js

```diff
@@ -0,0 +1,41 @@
+import React, { useRef, useState } from 'react'
+import { Button } from '../components/Button'
+import { QuestionHeading } from '../components/Typography'
+
+const resumeStyle = {
+  display: 'flex',
+  alignItems: 'center'
+};
+
+const ResumeUploadBtn = () => {
+  const [hint, setHint] = useState(null)
+  const inputFile = useRef(null);
+
+  const handleClick = () => {
+    inputFile.current.click();
+  }
+
+  const handleChange = (e) => {
+    if (e.target.value) {
+      setHint(<span>{e.target.value.match(/[\/\\]([\w\d\s\.\-\(\)]+)$/)[1]}</span>)
+    }
+
+    else {
+      setHint(<span style={{ color: '#F18383' }}>Please upload your resume</span>)
```

Comment:
Should be using theme colours here. Here's an example of how to do that:
https://github.com/nwplus/livesite/blob/8a746ca40a131660faf90761ea5890f35706c05f/src/components/Common.js#L110-L113

Reply:
Also can we move the span in hint to the `return ()` instead of including it with hint?

Reply:
Thanks for the comments! I've moved the spans to the return so that the hint is only string values. I've also replaced the span for the error mssg to use the TextInputLikeErrorMsg component! Is there another component that I should be using for the non-error message (ie. the resume file name) instead of span?